### ISSUE TITLE
genetic training and test (Unseen) set prediction using NEAT

### ISSUE THREAD

**Issue by Kuselokusi** [CodeReclaimers/neat-python#154]
Good-day,
I am currently using NEAT to make predictions on a categorical variable (with response true or false represented as 1 and 0) given a particular set of predictor variables. I have split the data into a training and test sets (75% and 25% respectively). So after genetic training takes place on training set using NEAT to a specified fitness level (specified in config file). Thereafter, predictions take place on the test set. My data is split relatively equally on the trues and falses in training. However, my true positive rate (TPR) is much higher than my true negative rate (TNR) on the test set (about 95% and 75% respectively). I have tried changing the fitness level however the TNR does not seem to be getting better. I need to get the TNR to around 85%. Please assist by suggesting something that could make the model better at identifying the falses correctly. Thank you in advance.
